### ISSUE TITLE
RFE for lazy maintainers ... how many days is the package in testing?

### ISSUE THREAD

**Issue by mcepl** [fedora-infra/bodhi#2176]
Unfortunately, most of packages I maintain has very low number of testers so (especially for EPEL branches) I usually end with waiting until the 7 (or 14) days expires and I can push it to the stable. Therefore, the following command is pretty common for me:
```
 $ bodhi updates query --status testing --user mcepl
ldapvi-1.7-29.fc26                    rpm    enhancement  testing   2018-02-19
ldapvi-1.7-29.fc27                    rpm    enhancement  testing   2018-02-19
ldapvi-1.7-29.el7                     rpm    newpackage   testing   2018-02-19
vim-vimoutliner-0.4.0-8.el7           rpm    enhancement  testing   2018-02-17
youtube-dl-2018.02.08-2.el6           rpm    enhancement  testing   2018-02-12
youtube-dl-2018.02.08-2.el7           rpm    enhancement  testing   2018-02-13
gen-oath-safe-0.10.1-3.el7            rpm    enhancement  testing   2018-02-07
youtube-dl-2016.12.09-1.fc23          rpm    enhancement  testing   2016-12-13
8 updates found (8 shown)
 $
```
(BTW, there used to be parameter ``--mine`` as an alias for ``--user=$ME``, wasn't there?)
(and yes, I am afflicted by #752 as well :( )

For lazy bums like myself, it would be enhancement, if I could have in the end number of days the package is in the state it is, e.g., something like:
```
ldapvi-1.7-29.el7                     rpm    newpackage   testing   2018-02-19 (7)
```

**Comment by bowlofeggs** [comment]
On 02/27/2018 04:02 AM, Matěj Cepl wrote:
> (BTW, there used to be parameter |--mine| as an alias for |--user=$ME|,
> wasn't there?)

```--mine``` is present in Bodhi >= 2.10.0:

https://bodhi.fedoraproject.org/docs/user/release_notes.html#id56

> For lazy bums like myself, it would be enhancement, if I could have in
> the end number of days the package is in the state it is, e.g.,
> something like:
> 
> |ldapvi-1.7-29.el7 rpm newpackage testing 2018-02-19 (7) |

Good idea, thanks for the RFE!

**Comment by mattiaverga** [comment]
I can work on this.
But we need at least a 4 (or 5?) chars column for that, and the line is already at 80 chars:
```
ldapvi-1.7-29.el7                     rpm    newpackage   testing   2018-02-19 (7)
ldapvi-1.7-29.el7                     rpm    newpackage   testing   2018-02-19(14)
```
Should I take the space from the name column? Or we can overpass the 80chars limit?
In my opinion the best option would be to remove the update type column ('newpackage') and use those 11 chars for the new column + more space for the name of the update. I don't think it's essential to have that column in the brief output.

**Comment by mcepl** [comment]
Either that, or just abbreviate (``new``, ``bug``, ``sec``)?

**Comment by bowlofeggs** [comment]
@mattiaverga Greetings! I think the value in the type of update might simply be to know whether it is a security update or not. We could probably use a ```*``` somewhere on the line to indicate security updates and reduce it down to one column. I did that for the ```bodhi composes list``` command and I think it works well there. What do you think?

**Comment by mattiaverga** [comment]
That sounds good. I'll edit my PR to work like that.

**Comment by bowlofeggs** [comment]
This patch is planned to be included in the upcoming [3.8.0 release](https://bodhi.stg.fedoraproject.org/docs/user/release_notes.html). You can get a copy of the RPM from my [bodhi-pre-release Copr project](https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release/).
